docs/astro/sql/operators/merge.rst

```diff
@@ -0,0 +1,143 @@
+.. _merge_operator:
+
+======================================
+merge operator
+======================================
+
+When to use the ``merge`` operator
+----------------------------------
+
+Merge operator is used when you expect some conflicts while merging two tables due to underline constraints. If you don't expect any conflicts you can use :ref:`append_operator`. There are two common cases when merging tables.
+
+Prerequisite
+------------
+
+Merge operator internally run different SQL queries based on the databases and some databases only allow you to run these queries if there are constraints on columns specified in parameter ``target_conflict_columns``. Below is the list of databases and their constraint requirement.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Database
+     - Constraint required
+   * - Bigquery
+     - No
+   * - Postgres
+     - Yes
+   * - Snowflake
+     - Yes
+   * - SQLite
+     - Yes
```

Comment:
Doesn't SQLite expect an index as opposed to constraint?

Reply:
@tatiana I think it's sqlite way of creating constraints, based on the following link 
https://stackoverflow.com/questions/15497985/how-to-add-unique-constraint-to-existing-table-in-sqlite